spec/spec.yml

```diff
@@ -4990,8 +4990,26 @@ definitions:
           - 'TEAM'
         groupId:
           type: string
-          format: uuid
+          format: uui
+        membershipStatus:
+          type: string
+          enum:
+          - 'REQUESTED'
+          - 'INVITED'
+          - 'APPROVED'
     - $ref: '#/definitions/UserRole'
+  UserGroupRoleWithRelated:
+    type: object
+    allOf:
+    - $ref: '#/definitions/UserGroupRole'
+    - type: object
+      properties:
+        platformName:
```

Comment:
Just occurred to me - why do we need 3 names instead of a single 'groupName' field?

Reply:
This currently reflects the endpoint. It will be updated along with a future PR in RF that will update the return of the endpoint. Mark as resolved.